packages/electron-builder/src/codeSign.ts

```diff
@@ -221,16 +221,12 @@ export async function findIdentity(certType: CertType, qualifier?: string | null
     for (const prefix of appleCertificatePrefixes) {
       checkPrefix(identity, prefix)
     }
-    const result = await _findIdentity(certType, identity, keychain)
-    if (result == null) {
-      throw new Error(`Identity name "${identity}" is specified, but no valid identity with this name in the keychain`)
-    }
-    return result
+    return await _findIdentity(certType, identity, keychain)
```

Comment:
if `this.platformSpecificBuildOptions.identity` is specified explicitly, we must throw error. Currently, we will throw error in the macPackager only for `mas`. If check removed from `findIdentity`, here, in the macPackager, we must add corresponding check.